zkevm-circuits/src/evm_circuit/execution/create.rs

```diff
@@ -177,23 +185,40 @@ impl<F: Field> ExecutionGadget<F> for CreateGadget<F> {
             );
         });
 
-        let transfer = TransferGadget::construct(
-            cb,
-            from_bytes::expr(&caller_address.cells),
+        // TODO：check for address collision case by code hash previous
```

Comment:
Remove todo？

Reply:
removed in [52be462](https://github.com/scroll-tech/zkevm-circuits/pull/406/commits/52be46292621459896972f3cac9d66df191894a9) thanks!